home.html

```diff
@@ -19,7 +19,43 @@ <h1>Good Morning, Alex.</h1>
     </header>
 
     <main class="main">
-      <section>Newsfeed</section>
+      <section class="newsfeed">
+          <ul class="newsfeed__list">
+            <li>
+              <div class="content-container">
```

Comment:
No need for this `div`. Just put those classes on the `li`. Also: please consider  a different class name as this is very generic (but long).